src/v3/src/mocks/browser.ts

```diff
@@ -29,9 +29,28 @@ export const getWorker = async (): Promise<SetupWorkerApi | null> => {
     const scenarioName = params.get('siw-mock-scenario') ?? DEFAULT_SCENARIO_NAME;
 
     // pass through mock response from query params
-    const mockResponseParam = params.get('siw-mock-response');
-    // eslint-disable-next-line import/no-dynamic-require, global-require
-    const mockResponse = mockResponseParam ? require(`./response${mockResponseParam}.json`) : undefined;
+    const responseName = params.get('siw-mock-response')
+      ?.replace(/^\//, '')
+      ?.replace(/\/$/, '');
+
+    let mockResponse;
+    try {
```

Comment:
I don't like this pattern at all. We should find a way to consolidate the mocks so we're not conditionally looking in multiple locations.